src/main/java/com/yurjinia/common/util/locale/CommonLocaleResolver.java

```diff
@@ -0,0 +1,31 @@
+package com.yurjinia.common.util.locale;
+
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import org.springframework.context.i18n.LocaleContextHolder;
+import org.springframework.web.servlet.LocaleResolver;
+
+import java.util.Locale;
+import java.util.Objects;
+
+public class CommonLocaleResolver implements LocaleResolver {
+
+    private static final Locale ukrainianLocale = new Locale("uk");
```

Comment:
all hardcoded occurences move to LangConstants class.

Reply:
also we don't want to put specific case within a variable.

we need to have an enum that stores information about all supported languages, and then do the check if request param has a specific language provided in enum. if yes - return new Locale with matched locale;

if not contains - return default one

at the end - return the request Locale.